include/mbedtls/ssl.h

```diff
@@ -2500,7 +2506,7 @@ const mbedtls_ssl_session *mbedtls_ssl_get_session_pointer( const mbedtls_ssl_co
  *
  *                      Note: The server uses its own preferences
  *                      over the preference of the client unless
- *                      MBEDTLS_SSL_SRV_RESPECT_CLIENT_PREFERENCE is defined!
+ *                      conf->respect_cli_pref is enabled!
```

Comment:
`conf->respect_cli_pref` is not part of the API so it should not be mentioned in the documentation. Suggested wording:

> By default, the server chooses its preferred ciphersuite among those that the client supports. If `mbedtls_ssl_conf_respect_client_preference` is called to prefer the client's preferences, the server instead chooses the client's preferred ciphersuite among those that the server supports.